omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -536,6 +536,8 @@
 default['private_chef']['oc_id']['sql_database'] = "oc_id"
 default['private_chef']['oc_id']['sql_user'] = "oc_id"
 default['private_chef']['oc_id']['sql_password'] = "snakepliskin"
+default['private_chef']['oc_id']['sql_ro_user'] = "oc_id_ro"
+default['private_chef']['oc_id']['sql_password'] = "look-but-don't-touch"
```

Comment:
Is this supposed to be sql_ro_password? This is setting sql_password a second time as is.